tests/system/test_streams.py

```diff
@@ -149,6 +161,12 @@ async def test_async_unary_stream_async_generator(async_echo):
 
     @pytest.mark.asyncio
     async def test_async_stream_unary_iterable(async_echo):
+        # NOTE: There are currently no plans for supporting async rest client-streaming.
```

Comment:
add link to bug: https://github.com/googleapis/gapic-generator-python/issues/2169